### DISCUSSION TITLE
Support for HTMX

### DISCUSSION THREAD

**Discussion by pspencer-arculus** [colinwilson/lotusdocs#36] [Feature Requests]
HTMX is an awesome reactive technology, and I would love to utilize it with your amazing docs theme! I would love to use hx- tags in the embedded HTML in my `docs/*.md`, like this example:
https://github.com/acaloiaro/hugo-htmx-go-template
The example includes a web server that serves the dynamic html.

I tried to inject HTMX into Lotus Docs similar to how `fetch-deps.sh` does it:
- clone lotus docs to themes/
- add HTMX script
https://github.com/acaloiaro/hugo-htmx-go-template/blob/main/bin/fetch-deps.sh
and I also tried adding the HTMX script tag into the head of Lotus Docs.

Thank you for considering this, and awesome work on this template! I love the theme, and the search bar.

**Comment by colinwilson** [comment]
Thank you for your suggestion.

I learnt about HTMX myself less than two weeks ago! I will definitely look at this more closely, tags in `*.md` files sounds very interesting.
